src/wdio/services/TerraCommands/chai-methods.js

```diff
@@ -88,9 +94,10 @@ function matchReference(matchType = 'withinTolerance') {
   );
 }
 
-const chaiMedthods = {
+const chaiMethods = {
   accessible,
+  getComparisonResults,
```

Comment:
anyone against me exporting the function for testing purposes? makes sense to me for isolation. but it does not fit `chai-methods`, since it's not an assertion.